examples/java-matter-controller/BUILD.gn

```diff
@@ -27,14 +27,7 @@ java_library("java") {
   ]
 
   sources = [
```

Comment:
I pulled your branch and tried to build the example project but I ended up getting build failures. I bisected between my change to support java-matter-controller on darwin, to this commit, and found the last good commit was 92d86d5afaa47446ebe6377d7350534da8ec3d84. You aren't having issues?

```
gen/third_party/connectedhomeip/src/controller/data_model/jni/OnOffClient-InvokeSubscribeImpl.cpp:136:142: error: implicit conversion loses integer precision: 'jint' (aka 'int') to 'uint16_t' (aka 'unsigned short') [-Werror,-Wimplicit-int-conversion]
```

Reply:
I didn't, and both Android CHIPTool and java-matter-controller builds passed in CI. I will let you know if I hit this error in future.